src/sql/src/plan/error.rs

```diff
@@ -275,6 +276,9 @@ impl PlanError {
             Self::InvalidKeysInSubscribeEnvelopeUpsert => {
                 Some("All keys must be columns on the underlying relation.".into())
             }
+            Self::InvalidKeysInSubscribeEnvelopeDebezium => {
+                Some("All keys must be columns on the underlying relation.".into())
```

Comment:
This is not covered, as it is distinct from the similar message bug in the context of an UPSERT.

Reply:
Yeah, similar to the EnvelopeUpsert I don't think it's possible to hit this code path today but `panic`ing or ignoring the error if it comes up seems worse than introducing this error path